src/main/java/pixelperfect/route/AsteroidNode.java

```diff
@@ -0,0 +1,24 @@
+package pixelperfect.route;
+
+/**
+ * An asteroid field Node which belongs in a Route. The asteroid field will be a dangerous location
+ * to fly in.
+ * 
+ * @author Floris Doolaard
+ *
+ */
+public class AsteroidNode extends RouteNode {
+
+  /**
+   * The constructor for the AsteroidNode.
+   * 
+   * @param summary
+   *          , a summary of the AsteroidNode.
+   * @param description
+   *          , a description of the AsteroidNode.
+   */
+  public AsteroidNode(String summary, String description) {
+    super(summary, description);
```

Comment:
Shouldn't the summary and description be defined by the type of RouteNode, and therefore be set by the subclass itself rather than taking those attributes as parameters to the constructor? It would be weird if two AsteroidNodes exists with a different summary, or not?